packages/embark/src/cmd/cmd_controller.js

```diff
@@ -749,7 +749,7 @@ class EmbarkController {
         engine.registerModuleGroup("contracts");
         engine.registerModuleGroup("pipeline");
         engine.registerModuleGroup("tests", options);
-        engine.registerModulePackage('embark-deploy-tracker', {plugins: engine.plugins});
+        engine.registerModulePackage('embark-deploy-tracker', {plugins: engine.plugins, trackContracts: false});
```

Comment:
Just for clarification: This is a fix because contracts have been tracked by default, but shouldn't be?

Reply:
Yes, but for this is for tests only, because you obviously don't want to track contracts in the tests

Reply:
Thanks for clarifying, this wasn't obvious to me. Also just noticed you put most of the information in the PR description. I looked at the individual commit which is why I've missed it.